examples/heroku/README.md

```diff
@@ -0,0 +1,53 @@
+# Heroky deployment graphql-faker
+
+We are going to create a package and deploy it to Heroku.
+
+# Create your sdl schema
+
+The graphql `SDL` schema is the basis of graphql-faker. If you have the schema in `JSON`, you will need to convert it to `SDL`. We've added the utility to do that right here.
+
+## JSON to SDL
+
+```
+yarn convert-graphql-json
+```
+
+# Pre deploy
+
+First. Create your app on Heroku. You will need the Heroku app name, e.g. `graphql-faker-1`.
+
+You will need to be logged in to Heroku container registry on the command line.
+
+```
+heroku container:login
+```
+
+# Deploy
+
+The steps here are:
+
+* Create a graphql-faker image that includes the created graphql schema.
+* Deploy that image to the heroku image registry.
+* Run the image and test locally
+* Access the image at `https://<HEROKU APP NAME>/herokuapp.com/graphql`
+
+```
+cd examples/heroku
+docker build .
+docker run -d -p 9002:9002 <IMAGEID>
+docker tag <IMAGEID> registry.heroku.com/<HEROKU APP NAME>/web
+docker push registry.heroku.com/<HEROKU APP NAME>/web
+heroku ps:scale web=1
+```
+
+Shoutout to [graphql-ufc-api](https://github.com/jgcmarins/graphql-ufc-api) where we got the demo schema.
+
+# FAQ
+
+## How do I get the docker image ID?
+
+Run `docker image ls`.
+
+## I'm using a schema.graphql file but it's not working?
+
+A `graphql` file is similar but not the same as an `sdl` file. Try changing your `graphql` file to `sdl` and rebuild the image.
```

Comment:
Not sure I understand this one, Faker shouldn't care about file endings at all.
In all our previous examples we used `.graphql` and also faker creates `.graphql` files by default.
So for consistency, it's better to have `.graphql` everywhere.

Reply:
Hmm, ok. I added a standard graphql file with a `# comment`. And graphql-faker didn't start. This was the reason for the comment. After converting to sdl, it did work.

Reply:
I will remove this comment, but there may be a bug in standard graphql file processing.